conode/run_nodes.sh

```diff
@@ -61,10 +61,11 @@ shift $((OPTIND-1))
 [ "${1:-}" = "--" ] && shift
 
 if [ ! -x ./conode ]; then
-	echo "No conode executable found. Use \"go build\" to make it."
+	echo "No conode executable found in the current directory. Use \"go build\" to make it."
 	exit 1
 fi
 
+bindir=`pwd`
```

Comment:
I suppose that the `run_nodes.sh` and `conode` should be in the same directory, that means that the following line is more adapted:

https://github.com/dedis/cothority/pull/1833/files#diff-1829914d2357dde4f1e83f7eb1b60906R63

Which is in fact a revert of the changes from here:

https://github.com/dedis/cothority/commit/678c4cae70f3a24611f57de5f318e31e210e0fae#diff-1829914d2357dde4f1e83f7eb1b60906

Reply:
I noticed your change and I'll remove this change.